src/object_accessor.hpp

```diff
@@ -36,64 +36,8 @@
 #include <string>
 
 namespace realm {
-//
-// Value converters - template specializations must be implemented for each platform in order to call templated methods on Object
-//
-template<typename ValueType, typename ContextType>
-class NativeAccessor {
```

Comment:
A comment pointing at `CppContext` is the best idea I have (until concepts come along).